docs/sources/alerting/migrating-alerts/opt-in.md

```diff
@@ -20,9 +19,9 @@ We recommend that you backup Grafana's database. If you are using PostgreSQL as
 
 To enable Grafana alerts:
```

Comment:
The task identified in the H1 is "opt in..." - shouldn't this say "To opt in ..."?

Reply:
Updated.